js/screens/OnBoarding/styles.js

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet, Dimensions } from 'react-native';
+const { width, height } = Dimensions.get('window');
+import { fonts } from '../../config/styles';
+
+const styles = StyleSheet.create({
+  images: {
+    height: height * 0.29,
+    width: width * 0.55,
+  },
+  text: {
+    color: '#0072A0',
+    marginTop: -90,
+    paddingLeft: 40,
+    paddingRight: 40,
```

Comment:
I am sure you can come up with other properties to work with than paddingLeft/right : 40. try using flex box